zaza/charm_tests/octavia/setup.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2018 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Code for configuring octavia."""
+
+import base64
+import logging
+
+import zaza.utilities.cert
+import zaza.charm_lifecycle.utils
+import zaza.charm_tests.test_utils
+import zaza.charm_tests.glance.setup as glance_setup
+
+
+def add_amphora_image():
+    """Add Octavia ``amphora`` test image to glance."""
+    image_name = 'amphora-x64-haproxy'
+    glance_setup.add_image(
+        'http://tarballs.openstack.org/octavia/test-images/'
+        'test-only-amphora-x64-haproxy-ubuntu-xenial.qcow2',
```

Comment:
We will be able to publish a local image in object storage in serverstack we can use here as a default. But let's make this configurable.